src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization;
+
+namespace System.Text.Json
```

Comment:
This file under `S/T/J/Serialization` doesn't match the namespace (though on its own that isn't an issue). I think naming this file `JsonSerializer.cs` and putting this static helper on it is also a bit confusing.

Consider calling it `JsonSerializer.Helpers.cs` and move it to the `S/T/J` directory. Alternatively, just move it in the existing `JsonSerializer.cs` file in `S/T/J`.

Better yet move it into the existing `JsonHelpers.cs` file/class.

Reply:
Moved to `JsonHelpers.cs`.